### DISCUSSION TITLE
Enable author to edit other post not his own

### DISCUSSION THREAD

**Discussion by kuslahne** [danpros/htmly#824] [Ideas]
Hi guys,
I need to know if we can assign author role can edit other post? For example it  will bring benefits when an article can be edit by collaboration.  Maybe provide article information with latest updated too.

Also does  the framework support custom plugin?

Thank you,
Kus

**Comment by danpros** [comment]
Hello,

At the moment we have the following user roles:

- admin: has access to all the administration features
- editor: can publish and manage static pages/posts including the posts of other users
- author: can publish and manage their own posts

Currently it doesn't support plugins like other CMS in general, but you can use `functions.php` inside your theme folder and create a custom function there. Later you can call this custom function from your theme.
